recipes/vs2008_runtime/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: vs2008_runtime
+  version: "9.0.30729.6161"
+
+source:
+  fn: vcredist.exe
+  # Downloads from https://www.microsoft.com/en-us/download/details.aspx?id=26368
+  url: https://download.microsoft.com/download/5/D/8/5D8C65CB-C849-4025-8E95-C3966CAFD8AE/vcredist_x86.exe  # [win32]
+  sha256: 6b3e4c51c6c0e5f68c8a72b497445af3dbf976394cbb62aa23569065c28deeb6                                  # [win32]
+  url: https://download.microsoft.com/download/5/D/8/5D8C65CB-C849-4025-8E95-C3966CAFD8AE/vcredist_x64.exe  # [win64]
+  sha256: b811f2c047a3e828517c234bd4aa4883e1ec591d88fad21289ae68a6915a6665                                  # [win64]
+
+build:
+  number: 0
+  skip: true          # [not win]
+  msvc_compiler: 9.0
+  script_env:
+    - CI
+
+test:
+  commands:
+    {% set libs = [
+        "msvcm90",
+        "msvcp90",
+        "msvcr90",
+        "vcomp90"
+    ] %}
+    {% for each_lib in libs %}
+    - if not exist "%PREFIX%\\{{ each_lib }}.dll" exit 1
+    - if not exist "%PREFIX%\\Library\\bin\\{{ each_lib }}.dll" exit 1
+    {% endfor %}
+
+about:
+  home: http://www.microsoft.com
+  license: Proprietary
```

Comment:
Is this OK?

Reply:
Yes. They tell us that we can redistribute the [runtime libraries]( https://msdn.microsoft.com/en-us/library/8kche8ah(v=vs.90).aspx ). Anything not in that list comes from this [Microsoft Redistribution]( https://www.microsoft.com/en-us/download/details.aspx?id=26368 ), which exists for this purpose.

Reply:
Also from a very old discussion that happened on this same topic would [highlight this comment]( https://github.com/conda-forge/staged-recipes/pull/809#issuecomment-226651627 ) as well.